libkbfs/tlf_journal.go

```diff
@@ -852,6 +853,8 @@ func (j *tlfJournal) removeFlushedBlockEntries(ctx context.Context,
 			storedBytesBefore, storedBytesAfter))
 	}
 
+	j.diskLimiter.onBlocksFlush(ctx, flushedBytes)
```

Comment:
"Ignored" blocks should also be treated as "flushed" from the quota tracker's perspective.  So you might need to plumb flushed byte counts back up from `ignoreBlocksAndMDRevMarkers`, otherwise the unflushed bytes counts in the tracker won't match with reality.